blog/docs/articles/knative-backstage-plugins.md

```diff
@@ -0,0 +1,62 @@
+# Knative and Backstage
+
+**Author: Ali Ok, Principal Software Engineer @ Red Hat**
+
+## What's the vision?
+
+[Backstage](https://backstage.io/) is a platform for building developer portals. It provides a unified way to manage and visualize the different resources that developers use in their daily work. 
```

Comment:
Adding `{:target="_blank"}` at the end of the link will make this page open in a new tab, so that blog post tab won't be lost!

Just a nit, to have a better reader experience!
```suggestion
[Backstage](https://backstage.io/){:target="_blank"} is a platform for building developer portals. It provides a unified way to manage and visualize the different resources that developers use in their daily work. 
```